sematic/ui/packages/main/src/pipelines/pod_lifecycle/PodLifecycle.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Box from '@mui/material/Box';
+import Chip from '@mui/material/Chip';
+import Typography from '@mui/material/Typography';
+import { usePipelinePanelsContext } from "src/hooks/pipelineHooks";
+import { useRunJobHistory } from 'src/hooks/podHistoryHooks';
+import { useRunPanelLoadingIndicator } from 'src/hooks/runDetailsHooks';
+import PodEventHistory from 'src/pipelines/pod_lifecycle/PodEventHistory';
+import styled from '@emotion/styled';
+import { theme } from '@sematic/common/lib/src/theme/mira';
+import CopyButton from '@sematic/common/lib/src/component/CopyButton';
+import { useCallback } from 'react';
+
+const StyledCode = styled.code`
+  font-size: 12px;
+  color: ${theme.palette.grey[500]};
+  cursor: text;
+  user-select: text;  
+`;
+
+// Separate component to assign key to the component, so that it will be re-mounted when runId changes
+function PodLifecycleWithRunId(prop: {runId: string}) {
+  const { runId } = prop;
+
+  const { value, loading } = useRunJobHistory(runId);
+
+  useRunPanelLoadingIndicator(loading);
+
+  const preventPropagation = useCallback((e: React.MouseEvent) => {
+    e.stopPropagation();
+  }, []);
+
+  if (!value) return null;
+  if (value?.length === 0) return <Typography variant="body2">No job history found</Typography>;
+
+  return <>
+    {value!.map(
+      (job, index) =>
```

Comment:
I think we should have the most recent event last, which means we'd need to reverse the order here.